numerapi/numerapi.py

```diff
@@ -734,11 +734,12 @@ def get_leaderboard(self, limit: int = 50, offset: int = 0) -> List[Dict]:
             utils.replace(item, "nmrStaked", utils.parse_float_string)
         return data
 
-    def stake_set(self, nmr) -> Dict:
+    def stake_set(self, nmr, username: str = None) -> Dict:
```

Comment:
please change that to `model_id`, similar to the other endpoints and make it a required parameter